library/Notifications/Api/V1/Contacts.php

```diff
@@ -0,0 +1,1050 @@
+        schema: new OA\Schema(ref: '#/components/schemas/ContactUUID')
+    )]
+    #[OA\Response(
+        response: 201,
+        description: 'Contact created',
+        content: new OA\JsonContent(
+            examples: [
+                'ContactCreated' => new OA\Examples(
+                    example: 'ContactCreated',
+                    ref: '#/components/examples/ContactCreated'
+                ),
+            ],
+            ref: '#/components/schemas/SuccessResponse'
+        )
+    )]
+    #[OA\Response(
+        response: 204,
+        description: 'Contact updated',
+    )]
+    #[OA\Response(
+        response: 400,
+        description: 'Bad request',
+        content: new OA\JsonContent(
+            examples: [
+                'InvalidRequestBody' => new OA\Examples(
+                    example: 'InvalidRequestBody',
+                    ref: '#/components/examples/InvalidRequestBody'
+                ),
+            ],
+            ref: '#/components/schemas/ErrorResponse'
+        )
+    )]
+    #[OA\Response(
+        response: 415,
+        description: 'Unsupported Media Type',
+        content: new OA\JsonContent(
+            examples: [
+                'ContentTypeNotSupported' => new OA\Examples(
+                    example: 'ContentTypeNotSupported',
+                    ref: '#/components/examples/ContentTypeNotSupported'
+                ),
+            ],
+            ref: '#/components/schemas/ErrorResponse'
+        )
+    )]
+    #[OA\Response(
+        response: 422,
+        description: 'Unprocessable Entity',
+        content: new OA\JsonContent(
+            examples: [
+                'GroupNotFound' => new OA\Examples(
+                    example: 'GroupNotFound',
+                    summary: 'Group not found',
+                    value: [
+                        'status'  => 'error',
+                        'message' => 'Group does not exist: X',
+                    ]
+                ),
+                'MissingRequiredRequestBodyField' => new OA\Examples(
+                    example: 'MissingRequiredRequestBodyField',
+                    ref: '#/components/examples/MissingRequiredRequestBodyField'
+                ),
+                'InvalidRequestBodyField' => new OA\Examples(
+                    example: 'InvalidRequestBodyField',
+                    ref: '#/components/examples/InvalidRequestBodyField'
+                ),
+                'InvalidIdentifier' => new OA\Examples(
+                    example: 'InvalidIdentifier',
+                    ref: '#/components/examples/InvalidIdentifier'
+                ),
+            ],
+            ref: '#/components/schemas/ErrorResponse'
+        )
+    )]
+    public function put(string $identifier, array $requestBody): array
+    {
+        if (empty($identifier)) {
+            throw HttpBadRequestException::create(['Identifier is required']);
+        }
+
+        $data = $this->getValidatedRequestBodyData($requestBody);
```

Comment:
This method only validates the given `$requestBody` and does not change it. It can be void. Please change the name to : `assertValidRequestBody(): void`